alethzero/MainWin.cpp

```diff
@@ -1635,6 +1635,13 @@ static shh::Topic topicFromText(QString _s)
 	return ret;
 }
 
+
+bool Main::sourceIsSolidity(std::string const& _source)
+{
+	// TODO: Improve this heuristic
+	return (_source.substr(0, 8) == "contract" || _source.substr(0, 2) == "/*");
```

Comment:
Ops ... I actually deleted a place where this check was happening. Will add it back in this function then.